src/main/java/duke/main/Ui.java

```diff
@@ -0,0 +1,100 @@
+package duke.main;
+
+import duke.task.Task;
+
+import java.util.Scanner;
+
+/**
+ * Represents an interface for user interaction. A <code>Ui</code> object is
+ * able to print and read user inputs, notifying Duke user of program status.
+ */
+public class Ui {
+
+    private String lastCommand;
+
+    private Scanner scanner;
+
+    public Ui() {
+        scanner = new Scanner(System.in);
+    }
+
+    public static void println(String string) {
+        System.out.println(string);
+    }
+
+    /**
+     * Reads a user command if it exists and returns to user.
+     *
+     * @return the user command
+     */
+    public String readCommand() {
+        if (scanner.hasNextLine()) {
+            lastCommand = scanner.nextLine();
+        }
+        return lastCommand;
```

Comment:
Since `lastCommand` is only used in this function, might want to move the private variable and make it a local variable inside this function.